content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+  Documents?` option active.
+- :guilabel:`Document Type`: select the credit note document type.
+- :guilabel:`Reversal Date`: type the date.
+
+Customer debit note
+~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer debit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a debit note. On the action within an invoice there is a option named
+:guilabel:`Debit note`, click on this action to be directed to the :guilabel:`Create credit note`
+form, then complete the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Copy lines`: mark checkbox to copy the invoice lines to the debit note.
+- :guilabel:`Debit note date`: type the date.
+
+.. note::
+   The document sequence (number) is brought from Uruware once the document has been processed.
+
+.. note::
+   The PDF of the validated document is pulled from Uruware following the specification by the
+   Uruguayan government (DGI).
+
+Addendas and disclosures
+========================
+
+*Addendas* and *disclosures* are additional notes and comments added to an electronic document that
+can be mandatory or optional. To create a new addenda, go to :menuselection:`Accounting --> Settings
+--> DGI --> Addendas and disclosures`.
+
+To create an addenda, the following information is required:
+
+- :guilabel:`Name`: Name of the Addenda or Mandatory Disclosure.
+- :guilabel:`Apply on`: Select the documents where the addenda or mandatory disclosure are
+  applicable.
+- :guilabel:`Type`: Select the type of remark, this will add it to the specific section in the XML.
+- :guilabel:`Content`: Add the complete text of the addenda or disclosure.
+- :guilabel:`Is disclosure?`: Select this box if the text is a mandatory disclosure, leave it blank
+  if it is additional information.
+
+Leyenda and additional information in product
+---------------------------------------------
+
+To add a leyenda or additional information to the product and XML, it is necessary to add the
+preconfigured addenda and disclosure to the product in the invoice line. The field called
+:guilabel:`Disclosure` will be used to add the leyenda to the product specified in the line.
+
+Leyenda and additional information in document
+----------------------------------------------
+
+To add a leyenda or additional information to the electronic document and XML, it is necessary to
+add the preconfigured addenda and disclosure to the document using the field :guilabel:`Addenda and
+Disclosures` that can be found in the *Other Info* tab in the document form view. The addenda and
+disclosures added here will appear in the XML and visibly in the PDF document.
```

Comment:
This applies for leyendas and additional information in documents + issuer, receiver and actual addendas. Just expand the explanation to all 4 of them.

Reply:
@samueljlieber rephrase section with: